cmd/clusterctl/pkg/client/config.go

```diff
@@ -47,19 +48,150 @@ func (c *clusterctlClient) GetProviderComponents(provider, targetNameSpace, watc
 	return components, nil
 }
 
+// GetClusterTemplateOptions carries the options supported by GetClusterTemplate.
+type GetClusterTemplateOptions struct {
```

Comment:
As mentioned [above](https://github.com/kubernetes-sigs/cluster-api/pull/2265/files#r376631963), we could just call this `ClusterTemplateOptions`.